typescript/src/offerings.ts

```diff
@@ -311,7 +311,7 @@ export interface PurchasesIntroPrice {
   /**
    * Unit for the billing period of the discount, can be DAY, WEEK, MONTH or YEAR.
    */
-  readonly periodUnit: string;
+  readonly periodUnit: "DAY" | "WEEK" | "MONTH" | "YEAR";
```

Comment:
This can also be `UNKNOWN` in unexpected scenarios:
```suggestion
  readonly periodUnit: "DAY" | "WEEK" | "MONTH" | "YEAR" | "UNKNOWN";
```